.changeset/rude-goats-destroy.md

```diff
@@ -0,0 +1,5 @@
+---
+"@boostxyz/sdk": patch
```

Comment:
this is not a patch, this would be a breaking change, which we should avoid at this point